src/config.ts

```diff
@@ -13,6 +13,94 @@ export const orientMap = {
   square: { height: 640, width: 640 },
 } as const
 
+/* AUTO-GENERATED STABLE HORDE MODELS */
+export const hordeModels = [
+  'colorbook',
+  'Papercutcraft',
+  'Xynthii-Diffusion',
+  'Redshift Diffusion',
+  'Mega Merge Diffusion',
+  'Borderlands',
+  'Voxel Art Diffusion',
+  'Elden Ring Diffusion',
+  'Cyberpunk Anime Diffusion',
+  'Future Diffusion',
+  'Vintedois Diffusion',
+  'Microworlds',
+  'mo-di-diffusion',
+  'Min Illust Background',
+  'Robo-Diffusion',
+  'stable_diffusion_inpainting',
+  'Dawgsmix',
+  'Zeipher Female Model',
+  'AIO Pixel Art',
+  'Dungeons and Diffusion',
+  'trinart',
+  'Yiffy',
+  'Midjourney PaintArt',
+  'App Icon Diffusion',
+  'Smoke Diffusion',
+  'Inkpunk Diffusion',
+  'Supermarionation',
+  'DnD Item',
+  'Zack3D',
+  'Fantasy Card Diffusion',
+  'stable_diffusion',
+  'Synthwave',
+  'Furry Epoch',
+  'Asim Simpsons',
+  'PortraitPlus',
+  'Anything Diffusion',
+  'Tron Legacy Diffusion',
+  'Clazy',
+  'Valorant Diffusion',
+  'Papercut Diffusion',
+  'Stable Diffusion 2 Depth',
+  'ChromaV5',
+  'stable_diffusion_2.1',
+  'Archer Diffusion',
+  'Guohua Diffusion',
+  'Dark Victorian Diffusion',
+  'Analog Diffusion',
+  'vectorartz',
+  'Classic Animation Diffusion',
+  'Eimis Anime Diffusion',
+  'Ranma Diffusion',
+  'Microscopic',
+  'Dreamlike Diffusion',
+  'GTA5 Artwork Diffusion',
+  'Poison',
+  'Midjourney Diffusion',
+  'Samdoesarts Ultmerge',
+  'kurzgesagt',
+  'Dreamlike Photoreal',
+  'Trinart Characters',
+  'Arcane Diffusion',
+  'JWST Deep Space Diffusion',
+  'RPG',
+  'Hassanblend',
+  'waifu_diffusion',
+  'ModernArt Diffusion',
+  'Darkest Diffusion',
+  'Balloon Art',
+  'Comic-Diffusion',
+  'BubblyDubbly',
+  'Eternos',
+  'Van Gogh Diffusion',
+  'Double Exposure Diffusion',
+  'Squishmallow Diffusion',
+  'Funko Diffusion',
+  'Hentai Diffusion',
+  'Ghibli Diffusion',
+  'Seek.art MEGA',
+  'Knollingcase',
+  'ACertainThing',
+  'Spider-Verse Diffusion',
+  'Wavyfusion',
+  'Nitro Diffusion',
+] as const
+/* END OF AUTO-GENERATED STABLE HORDE MODELS */
```

Comment:
Why not export the automatically generated part as a separate json file? This reduces the possibility of losing sync due to mistakes.

Reply:
But the `Config` should always be synchronous, isn't it? It would be asynchronous if you add any file system APIs.

Reply:
I mean you `require` the generated JSON.

Reply:
Ah, that makes sense too.